ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -52,6 +58,13 @@ object Config {
     CommandConfiguration.DefaultTrackerRetentionPeriod
 
   val DefaultMaxInboundMessageSize: Int = 64 * 1024 * 1024
+  val DefaultIndexerInputMappingParallelism: Int = 16
```

Comment:
Why are the application defaults so different to the defaults at the `IndexerConfig.scala` level